config.go

```diff
@@ -29,7 +30,8 @@ type Env struct {
 }
 
 type Options struct {
-	Scale *ScaleOptions `yaml:"scale,omitempty"`
+	Scale     *ScaleOptions     `yaml:"scale,omitempty"`
+	Resources *ResourcesOptions `yaml:"resources,omitempty"`
```

Comment:
I don't think these should be pointers.  The zero value of the struct should probably be sufficient?